packages/block-library/src/column/edit.js

```diff
@@ -73,6 +76,10 @@ function ColumnEdit( {
 			: InnerBlocks.ButtonBlockAppender,
 	} );
 
+	const columnsCount = columnsIds.length;
+	const currentColumnPosition = columnsIds.indexOf( clientId ) + 1;
```

Comment:
Would getBlockIndex and getBlockOrder work for this? When I tried earlier this week I was not able to solve it.